ultralytics/yolo/engine/model.py

```diff
@@ -78,7 +79,7 @@ def __init__(self, model: Union[str, Path] = 'yolov8n.pt', task=None) -> None:
             task (Any, optional): Task type for the YOLO model. Defaults to None.
 
         """
-        self._reset_callbacks()
+        self.callbacks = deepcopy(callbacks.default_callbacks)
```

Comment:
@dh031200 do we require deepcopy() here? Can you we simply use the dictionary built-in .copy() method, i.e. here is my proposed change:
```diff
- from copy import deepcopy
- self.callbacks = deepcopy(callbacks.default_callbacks)
+ self.callbacks = callbacks.default_callbacks.copy()
```